.github/workflows/integration_test.yml

```diff
@@ -53,6 +52,26 @@ jobs:
       - name: Install poetry dependencies
         run: poetry install
 
+      - name: Fetch latest rippled amendments
+        if: ${{ github.event_name == 'pull_request' }}
+        run: |
+          poetry run python tools/fetch_rippled_amendments.py
+
+      - name: Add and commit rippled.cfg
```

Comment:
Why run this as a part of integration tests rather than have a separate PR open on a cron job to update it?

Reply:
Integration tests make use of a docker container. The `rippled.cfg` file is used to kick-start the rippled server with custom amendments. The results of the integration tests differ based on which amendments have been enabled in the standalone mode.

> separate PR open on a cron job to update it

Are you suggesting Github Actions cron jobs (or) running a cron-job from my personal machine to periodically fetch rippled PRs ?

Reply:
A GitHub Action cron job.